docs/topics/Data-Verification.md

```diff
@@ -0,0 +1,62 @@
+# Data Verification
+
+Submitted tournaments run through a series of manual and automated checks before they may be processed for rating calculations. The [](DataWorkerService.md) and [](Automated-Checks.md) pages describe most of the automatic flow, while this page details the manual flow for verifying tournament data and provides more transparency on various conditions. 
+
+## Verifying or rejecting data
+
+Once data finishes running through automated checks, each game score, game, match, and tournament will be marked as either `PreVerified` or `PreRejected`. Typically, `PreVerified` data will end up `Verified` and `PreRejected` data will end up `Rejected` after manual review, and the steps below explain how the o!TR team interprets or modifies these statuses.
+
+<procedure>
+<step>
+First, general information about the tournament (rank limit, lobby size, any gimmick) is cross-checked against the tournament's wiki page or forum post. If the tournament does not meet the <a href="Tournament-Approval.md#acceptance-criteria">acceptance criteria</a>, it will be rejected, meaning that none of the match data will factor into ratings.
```

Comment:
```suggestion
First, general information about the tournament, such as rank limit, name, and abbreviation, is cross-checked against the tournament's wiki page or forum post. If the tournament does not meet the <a href="Tournament-Approval.md#acceptance-criteria">acceptance criteria</a>, it will be rejected, meaning that none of the match data will factor into ratings.
```